gst/nnstreamer/tensor_query/tensor_query_serversink.c

```diff
@@ -302,7 +302,10 @@ gst_tensor_query_serversink_render (GstBaseSink * bsink, GstBuffer * buf)
 
   meta_query = gst_buffer_get_meta_query (buf);
   if (!meta_query) {
-    nns_logw ("Cannot get tensor query meta. Drop buffers!");
+    nns_loge ("Cannot get tensor query meta. Drop buffers!\n"
+        "There are elements that are not available on the query server.\n"
+        "Please check available elements on the server."
+        "See: https://github.com/nnstreamer/nnstreamer/wiki/Available-elements-on-query-server");
     return GST_FLOW_OK;
```

Comment:
If meta is not available, you cannot reply to clients, right?

Then,... let's generate GST_FLOW_${error} if it fails to get meta repeatedly without having a frame with meta.

E.g., 
```
static int metalessFrameCount = 0;
...
if (meta) {
   process normally;
   metalessFrameCount = 0;
} else {
   write error message;
   metalessFramecount ++;
   
   if (metalessFrameCount > limit) {
      write fatal error message;
      return GST_FLOW_${error};
}
...
return GST_FLOW_OK;
```

Reply:
> If meta is not available, you cannot reply to clients, right?

Right. If the client don't receive the reply within timeout, it will stop the playing.
I will change to stop the server if there is no meta in the buffer received from all clients.
Thanks.

Reply:
Yes. what we need to do is:

notify the server pipeline writer that he/she made a mistake in GSTMETA handling (an element that cannot handle GSTMETA is in the pipeline) as soon as possible and as clear as possible.

Reply:
And... this ```limit``` might need to be a property of tensor-query-server-sink with some reasonable default value.

Reply:
I changed it to compare using the number of connected client instead of `limit`.
Because, If the client doesn't receive the buffer from the server, It doesn't send the buffer to the server again.

Reply:
No. don't make it #clients.
What if #clients = 100 and only 1 client has actually sent a query?
In most cases, server pipeline writers will want to set limit = 1 unless the pipeline is supposed to create a few non-Meta buffers (to be wasted) along with with-Meta buffers from a single query input. (e.g., for each query-src input frame, three sink output frames are created and only one of the three has a valid Meta.)

Reply:
Ok. Default `metaless_frame_limit` value is set to 1 for general case.